subsys/testsuite/ztest/src/ztest_new.c

```diff
@@ -357,6 +388,7 @@ static int run_test(struct ztest_suite_node *suite, struct ztest_unit_test *test
 	run_test_functions(suite, test, data);
 out:
 	ret |= cleanup_test(test);
```

Comment:
Can we move `cleanup_test` after calling `after` rules so we have same sequence of events as `run_test` in `KERNEL` enabled builds?

Reply:
Done